### PR TITLE
mailgun takes email input to send confirmation email template

### PR DESCRIPTION
**Pull request by pvalderama**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const Mailgun = require('mailgun-js')
+
+const app = express();
+
+const api_key = '317a0f64411bd75d7598da6be916803b-46ac6b00-50f5a2ee';
```
Comment (allopez7) [inline app.js:6]: You will need to import from .env file, which is written like this require('dotenv').config(). Inside of .env make sure you create API_KEY= '317....'

File: app.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+const domain = 'sandbox47881c22500a4bf7a21f621e5bc25800.mailgun.org';
```
Comment (allopez7) [inline app.js:8]: also add domain to .env file as DOMAIN='sandbox....'

Comment (allopez7) [inline app.js:8]: this can also be added to .env to make things look cleaner

File: app.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+const from_who = 'pvalderama@gmail.com';
```
Comment (allopez7) [inline app.js:10]: and this

File: app.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    const fp = path.join(__dirname, 'template.txt')
+
+    const mailgun = new Mailgun({apiKey: api_key, domain: domain});
+
+    const data = {
+        from: from_who,
+        to: req.params.mail,
+        subject: 'Confirm E-mail for C0D3',
+        text: 'Confirm E-mail',
+        attachment: fp
```
Comment (allopez7) [inline app.js:24]: fp could be more descriptive

File: app.js (hunk 1 above)
Comment (songz) [reply app.js:6]: `process.env.MAIL_KEY`

Comment (songz) [reply app.js:6]: `./lib/mailer.js`

**Pull request closed without merging**